PrimalHQ/primal-android-app
# Accessibility incorrectly labeled buttons

Fix for https://github.com/PrimalHQ/primal-android-app/issues/76

The code could have been more compact but I have chosen clarity by creating new accessibility_* strings rather than using shortcuts.

### To test or develop accessibility:

Android studio:

- Select an emulator device that has Google Play Store available (eg Google Pixel 7 pro).
- Use an android image that has play store built in (eg Android 14).
- Use google play store on the emulator device to install Google Android Accessibility Suite.
- Turn on talkback in the settings. Make sure to enable the hovering shortcut so that you can toggle Talk Back on/off quickly.
